manifests/init.pp

```diff
@@ -145,6 +149,11 @@
   Optional[Stdlib::Absolutepath]                 $db_tls_cert_file       = undef,
   Optional[Stdlib::Absolutepath]                 $db_tls_key_file        = undef,
   Optional[Stdlib::Absolutepath]                 $db_tls_cacert_file     = undef,
+  Optional[Hash[Enum['max_connections',
```

Comment:
complex enough to be its own custom datatype. but we can do this later, just wanted to point this out

Reply:
check